### PR TITLE
Currency format

### PR DESCRIPTION
**Pull request by SantiDepetris**
Description: Pass the format, to Argentina format 

| Tables        | Are           |
| ------------- |:-------------:|
| image | image |

### CODE REVIEW

File: src/components/Bank.js [hunk 1]
```diff
@@ -2,20 +2,26 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styles from '../styles/Bank.module.scss';
 
-export default function Bank({ bank: { name, sell, buy } }) {
+export default function Bank({ bank: { name, sell, buy, url } }) {
   const noData = hasExchange(sell, buy) ? '' : `${styles.no_data}`;
   const mainClasses = `${styles.box} ${noData}`;
+
+  console.log('Precio de Venta: ' + sell);
```
Comment (edudepetris) [inline src/components/Bank.js]: remove these logs

File: src/components/Bank.js [hunk 2]
```diff
@@ -33,10 +39,17 @@ function hasExchange(sell, buy) {
   return true;
 }
 
+function currencyFormat(price) {
+  const parsePrice = Number.parseFloat(price).toFixed(2);
```
Comment (edudepetris) [inline src/components/Bank.js]: great work, but we could use the lang to do it for us.

```js
const number = 12345.67;
number.toLocaleString('es-AR', {currency: 'ARS', style: 'currency'});
// result "$ 12.345,67"
```

If you wonder how `toLocaleString` works check it here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString.
Under the hood it's calling https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat

**Review by edudepetris — CHANGES_REQUESTED** [review]
Good work! 

It will required some changes, I put it as comments.

I've notice this PR was created on the top of a previous one. I mean, you're sending two pr here. Please create a solo pr for solve this problem.

File: src/components/Bank.js (hunk 1 above)
Comment (SantiDepetris) [reply src/components/Bank.js]: Done

File: src/components/Bank.js (hunk 2 above)
Comment (SantiDepetris) [reply src/components/Bank.js]: Thanks, will do

**Review by edudepetris — CHANGES_REQUESTED** [review]
@SantiDepetris i noticed if you run storybook the url is missing there
`yarn storybook` 

could you add any url to `Bank.stories.js`

**Comment by edudepetris** [comment]
@SantiDepetris notice that you'll need to resolve Bank.js conflicts

**Review by edudepetris — APPROVED** [review]
Great work! 🎉

**Pull request merged**